neurodsp/sim/periodic.py

```diff
@@ -303,3 +310,15 @@ def get_burst_samples(is_oscillating, fs, freq):
     bursts = np.repeat(is_oscillating, n_samples_cycle)
 
     return bursts
+
+
+def _check_tiling(fs, freq):
+    """Check if tiling can produce full oscillations."""
+
+    if not (fs/freq).is_integer():
+
+        warnings.warn('''
+        The requested frequency and sampling rate are not evenly divisible, resulting
+        in an artifactual spectral slope. Consider updating freq and fs to an integer
+        divisor/multiple pair.
+        ''', category=RuntimeWarning)
```

Comment:
I think we could perhaps add the middle step to this warning, which is that not evenly divisible -> uneven number of cycles -> a potentially unexpected power spectrum since I'm not sure mentioning an 'artifactual spectral slope' is obvious if one doesn't already know what's going on (and to nitpick - it's not strictly "artifactual" in that it is the correct / expected slope for a non integer number of cycles. 

Suggested re-wording:
"The settings for the frequency and sampling rate are not evenly divisible, meaning there is not an integer number of cycles for the requested simulated oscillation. In the frequency domain, this can lead to power in non-simulated frequencies in the power spectrum. You may want to consider updating freq and fs to an integer divisor/multiple pair."

^This is perhaps a bit long though, and could be condensed?

Reply:
Thanks! I like you re-wording - I'll condense it a bit.